app/views/quotas/search.html.erb

```diff
@@ -2,6 +2,10 @@
 
 <h1>Quota <%= @quota_definition.quota_order_number_id %></h1>
 
+<% flash.each do |key, value| %>
```

Comment:
Flashes should already be getting shown? If thats not working then the existing implementation should be corrected? That being said your screen grab shows the govuk formatted flash message from `_flashes.html.erb` so I think it is this block which is not rendering for some reason. I'd expect if you drop this it'll work just the same?

https://github.com/trade-tariff/trade-tariff-admin/blob/main/app/views/application/_flashes.html.erb

https://github.com/trade-tariff/trade-tariff-admin/blob/main/app/views/layouts/application.html.erb#L55

Reply:
👍 yep, that's right. I've removed that and it still works.
(I copied and pasted from the backend)

Removed!